stacker/plan.py

```diff
@@ -235,16 +241,10 @@ def execute(self):
         submit them in parallel based on their dependencies.
         """
 
-        attempts = 0
         try:
             while not self.completed:
-                attempts += 1
-                if not attempts % 10:
-                    self._check_point()
```

Comment:
We should try and keep this - logging every attempt gets really noisy, since CF isn't that fast.